manage.py

```diff
@@ -6,7 +6,10 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'demo.settings')
+    env = os.environ.get('ENVIRONMENT', 'development')
```

Comment:
@camilamaia I suggest adding an environment variable called `ENVIRONMENT: production` in production. So that `env = os.environ.get('ENVIRONMENT', 'development')` can be `production` thus the setting can be `demo.settings.production`